learn-ocaml.el

```diff
@@ -170,6 +179,14 @@ Function added in the `kill-emacs-query-functions' hook."
       (ignore-errors (delete-directory (learn-ocaml-temp-dir)))))
   t)
 
+(defun learn-ocaml-server-config (json)
+  "Set the global variable learn-ocaml-use-pswd according
+to the boolean contained in the json returned by the client"
+  (if (eql (cdr (assoc 'use_passwd (json-read-from-string json)))
+         :json-false)
+    (setq learn-ocaml-use-pswd nil)
+  (setq learn-ocaml-use-pswd t)))
```

Comment:
Change here also and everywhere in the file: `pswd` → `passwd`

BTW the logics seems broken here: if `json` does not contain `{"use_passwd": …}`, then the `eql` returns nil, then it is assumed use_passwd=true ! although it is maybe not supported.
So you should probably test with `:json-true` and switch the two `if` branches?